source/configuration_manual/namespace/index.rst

```diff
@@ -555,109 +563,156 @@ on and setting ``subscriptions = no`` for the other namespaces:
+Settings
+========
+
+.. _setting-namespace-mailbox_auto:
+
+``namespace/mailbox/auto``
+--------------------------
 
-The mailbox section name specifies the mailbox name. If it has spaces, you can
-put it in `quotes`. The mailbox settings are:
+- Default: ``no``
+- Values:  ``create``, ``no``, or ``subscribe``
 
-* auto: Autocreate/subscribe mailbox?
+Autocreate and/or subscribe to the mailbox?  Options:
 
- * no: Neither
- * create: Autocreate, but don't autosubscribe
- * subscribe: Autocreate and autosubscribe
+============== ==================================
+Value          Description
+============== ==================================
+``create``     Autocreate but don't autosubscribe
+``no``         Don't autocreate or autosubscribe
+``subscribe``  Autocreate and autosubscribe
+============== ==================================
+
+Autocreated mailboxes are created lazily to disk only when accessed for the
+first time. The autosubscribed mailboxes aren't written to subscriptions file,
+unless SUBSCRIBE command is explicitly used for them.
 
-* special_use: Space-separated list of SPECIAL-USE flags to use for the
-  mailbox. There are no validity checks, so you could specify anything you want
-  in here, but it's not a good idea to use other than the standard ones
-  specified in the RFC.
 
-  .. Note::
+.. _setting-namespace-mailbox_autoexpunge:
 
-    Due to a bug in Dovecot v2.2.30+ if special-use flags are used, SPECIAL-USE
-    needs to be added to post-login CAPABILITY response as RFC 6154 mandates. You
-    can do this with imap_capability = +SPECIAL-USE
+``namespace/mailbox/autoexpunge``
+---------------------------------
 
-.. _namespaces-autoexpunge:
+.. versionadded:: v2.2.20
 
-* autoexpunge=<time>: Automatically at user deinitialization expunge all mails in this mailbox whose
-  saved-timestamp is older than `<time>` (e.g. autoexpunge=30d). This removes the
-  need for :ref:`plugin-expire` if you
-  don't care that the expunging may not always happen in time.
+- Default: ``0``
+- Values:  :ref:`time`:
 
-  .. versionadded:: v2.2.20
+Expunge all mails in this mailbox whose saved-timestamp is older than this
+value.
 
-* For IMAP and POP3 this happens after the client is already disconnected.
-* For LMTP this happens when the user's mail delivery is finished. Note that if
-  there are multiple recipients this may delay delivering the mails to the
-  other recipients.
-* Also doveadm and other processes verify this, which may be unnecessary. So it
-  may be better to explicitly enable this only inside protocol imap, pop3 and
-  maybe lmtp. You can do this with
+For IMAP and POP3 this happens after the client is already disconnected.
 
-  Example:
+For LMTP this happens when the user's mail delivery is finished. Note that if
+there are multiple recipients this may delay delivering the mails to the
+other recipients.
 
-  .. code-block:: none
+doveadm and other processes use this as well. If you want to limit expunging
```

Comment:
That's not true. Autoexpunging is done only by imap, pop3 and lmtp. Nothing else.